docs/usage-tui.md

```diff
@@ -5,107 +5,106 @@ emulator.
+- `Wallet`: main tab, where you can do all the wallet operations
 - `Help`: a short help about `Knit` commands
-- `About`: an about screen, with licensing information
+- `About`: an about screen with licensing information
 - `Logs`: logs from an embedded Cardano node, which may be useful should something go wrong
 
 All widgets that have a lot of text (e.g. About and Logs) can be scrolled with mouse, arrow keys or
 `j`/`k` keys.
 
 ## Wallet tab &mdash; the heart of Ariadne
 
-On the `Wallet` tab you can see all your wallets with their accounts organized in a tree. To the
-right of the tree you can see details of the selected wallet or account.
+In the `Wallet` tab, you can see all your wallets and accounts organized as a tree. To the
+right of the tree, you can see details of the selected wallet or account.
 
-The tree can be navigated in several ways. First of all, you can select any entry by clicking it
-with mouse. You can also change the current selection with the keyboard. Up and Down keys will go to
-the previous and the next item respectively, including the `[ + Add wallet ]` button. Right key will
-go to the first child of the current item and Left key will go to the parent. Ctrl-Up and Ctrl-Down
-can be used to jump between wallets: Ctrl-Up will first go to the current wallet, when an account is
-selected, then to the previous wallet. Ctrl-Down will always go to the next wallet.  Finally, `h`,
+The tree can be navigated in several ways. First of all, you can select any entry by clicking on it
+with the mouse. You can also change the current selection with the keyboard. Up and Down keys will go to
+the previous and the next item respectively (including the `[ + Add wallet ]` button). Right key will
+go to the first child of the current item, and Left key will go to the parent. Ctrl-Up and Ctrl-Down
+can be used to jump between wallets: Ctrl-Up will first go to the current wallet (when an account is
+selected), then to the previous wallet. Ctrl-Down will always go to the next wallet. Finally, `h`,
 `j`, `k`, `l` keys will work exactly as Left, Down, Up and Right arrow keys (without Ctrl).
 
-To create a new wallet, select `[ + Add wallet ]` line in the tree, so that a special widget appears
-to the right. There you can either create a completely new wallet by specifying its name and a
+To create a new wallet, select the `[ + Add wallet ]` line in the tree. A special widget will appear
+on the right. There you can either create a completely new wallet by specifying its name and
 passphrase, or restore an existing one. When a new wallet is created, Ariadne will print you its
 mnemonic. **Please save it somewhere safe**.
 
-Note that Ariadne mnemonics have more words than 12-words mnemonics of Daedalus, and that the last
-word of the mnemonic is always fixed (currently it is `ariadne-v0`).  This means you won't be able
+Note that Ariadne mnemonics have more words (13) than mnemonics of Daedalus (12), and that the last
```

Comment:
@gromakovsky told me to remove "13" from earlier versions since user can ask for more entropy and thus get more words. Not sure how to proceed with this.
Also not sure whether daedalus can give more than 12.